web/src/lib/components/ui/add-resource/add-resource.svelte

```diff
@@ -35,28 +37,39 @@
       </div>
     {/each}
   </div>
+
   <div class="mx-7 flex flex-col">
-    {#if subjects.length > 0}
-      <div class="flex flex-row mb-7">
+    {#if subjects}
+      <div class="flex flex-row mb-7 items-baseline">
         <div class="my-5 flex w-40 text-xl font-bold">Fächer:</div>
-        <div class="flex flex-wrap bg-white rounded-md p-4 shadow-custom max-w-80">
-          {#each subjects as subject, i}
-            <div class="flex items-center space-x-2 mx-1">
-              <Checkbox class="m-1" bind:checked={selectedSubjects[i].selected} />
-              {subject.name}
-            </div>
-          {/each}
-        </div>
+        {#if subjects.length > 0}
+          <div class="flex flex-wrap bg-white rounded-md p-4 shadow-custom max-w-80">
+            {#each subjects as subject, i}
+              <div class="flex items-center space-x-2 mx-1">
+                <Checkbox class="m-1" bind:checked={selectedSubjects[i].selected} />
+                {subject.name}
+              </div>
+            {/each}
+          </div>
+        {:else}
+          <div class="text-lg font-semibold">Keine Fächer vorhanden.</div>
+        {/if}
```

Comment:
You don't need the if/else branching here. You can do `{#each subjects as subject, i} ... {:else} Keine Fächer vorhanden`